packages/mdc-list/mdc-list.scss

```diff
@@ -90,23 +90,44 @@
   @include mdc-rtl-reflexive-property(margin, auto, 0, ".mdc-list-item");
 }
 
-.mdc-list-item__text,
-.mdc-list-item__secondary-text {
+.mdc-list-item__text {
   @include mdc-typography-overflow-ellipsis;
 
+  align-self: flex-start;
+}
+
+$mdc-list-item-primary-text-baseline-height_: 32px;
+$mdc-list-item-secondary-text-baseline-height_: 20px;
+
+$mdc-list-dense-item-primary-text-baseline-height_: 24px;
+$mdc-list-dense-item-secondary-text-baseline-height_: 20px;
```

Comment:
if the dense version is the same as the regular version...then you probably dont need it?